### DISCUSSION TITLE
Explanation of Subscriptions (via Notifications)

### DISCUSSION THREAD

**Discussion by CalamityAdam** [community/community#23753] [New to GitHub]
<p>Can someone please explain a few things about the Subscriptions page please.<br>
I would like some more info around the following points:</p>
<ul>
<li>What are Subscriptions?</li>
<li>Why am I automatically subscribed to thousands of Pull Requests having never voluntarily subscribed to a single repository/pull request?</li>
<li>How do I stop being automatically subscribed to repositories and Pull Requests?</li>
<li>What is the ideal usage of a subscription? In what use case would the <a href="http://github.com/notifications/subscriptions" rel="nofollow noopener">github.com/notifications/subscriptions</a> page bring value to me as a developer that the notifications page does not already bring?</li>
<li>Most importantly, how do I unsubscribe from <strong>all</strong> in a bulk action?</li>
</ul>
<p>bonus: Why am I limited to only selecting 25 subscription at a time being that there are thousands of subscriptions set automatically?</p>
<p>Thank you!</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi calamityadam! :wave:</p>
<ul>
<li>Subscriptions aren’t new, exactly, they’re just giving you visibility into the places that you’re already set up to get notifications from.</li>
<li>There are a bunch of ways that you could be set up to receive notifications about a repo or PR that don’t involve you manually subscribing yourself, such as being assigned to a PR, being <span class="mention">@mentioned</span>, etc. More details <a href="https://help.github.com/en/github/managing-subscriptions-and-notifications-on-github/about-notifications#default-subscriptions">can be found here</a>.</li>
<li>There is a setting that automatically sets you to watch repositories as you gain access to them. You can <a href="https://help.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications#automatic-watching">read about disabling it here</a>. That wouldn’t stop you being subscribed to things you are <span class="mention">@mentioned</span> in or assigned to, though. It is possible to <a href="https://help.github.com/en/github/managing-subscriptions-and-notifications-on-github/managing-your-subscriptions#unwatch-a-repository">“ignore” a repository when unwatching it</a> so that you wouldn’t receive notifications when <span class="mention">@mentioned</span> or assigned to issues or PRs, but obviously you need to be very certain that missing those notifications won’t cause you problems.</li>
<li>The subscriptions page allows you to sort by the date the subscription started, so you can potentially more easily review old and out of date subscriptions, or only your most recent ones. It also shows source of notifications that won’t show up in the notifications view if you have marked them “done”. <a href="https://help.github.com/en/github/managing-subscriptions-and-notifications-on-github/managing-your-subscriptions#benefits-of-unsubscribing-from-the-subscriptions-page">More details here.</a>
</li>
<li>Unsubscribing from absolutely everything isn’t possible through the web UI, sorry! You could perhaps <a href="https://developer.github.com/v3/activity/notifications/">use the API</a> to do things faster than 25 at a time, though.</li>
</ul>
<p>Bonus - I suspect this is about performance - if we tried to load all of your subscriptions through the web UI in one go it would likely time out and cause problems.</p>
<p>Sorry that these aren’t exactly the answers you want to hear! We’re iterating on the notifications system pretty quickly at the moment, so you could <a href="https://support.github.com/contact/feedback">submit any ideas for improvements you might have through our official product feedback form</a> so that our product team can see exactly how you’d like this feature to work!</p>
